src/backend/api/meals.js

```diff
@@ -3,12 +3,68 @@ const router = express.Router();
 const knex = require('../database');
 
 router.get('/', async (req, res) => {
+  let meals = knex.select('meal.*').from('meal');
+
+  if ('maxPrice' in req.query) {
+    meals = meals.whereBetween('price', [0, req.query.maxPrice]);
```

Comment:
I think this works but it could be simplified
```suggestion
    meals = meals.where('price', '<=', req.query.maxPrice);
```